### PR TITLE
Completed javascript-loops

### PR DESCRIPTION
**Pull request by amorabryan**
image

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  } return repeated;
+}
```
Comment (kurtyazdizadeh) [inline javascript-loops/main.js]: This is returning one less times than it should

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
One small change! Almost got it

File: javascript-loops/main.js (hunk 1 above)
Comment (amorabryan) [reply javascript-loops/main.js]: So for line 31 it would be `(count <= times)`

**Review by kurtyazdizadeh — APPROVED** [review]
Approved!

**Pull request merged**